src/Features/Core/Portable/Completion/Providers/AbstractSymbolCompletionProvider.cs

```diff
@@ -115,6 +130,7 @@ private bool ShouldIncludeInTargetTypedCompletionList(ISymbol symbol, ImmutableA
                                select g;
 
             var itemListBuilder = ImmutableArray.CreateBuilder<CompletionItem>();
+            var typeConvertibilityCache = new Dictionary<ITypeSymbol, bool>(AllNullabilityIgnoringSymbolComparer.Instance);
```

Comment:
can you comment why this is the right comparer?

Reply:
I don't think I need this comparer anymore since I'm guaranteeing the type symbols as keys will not have nullability info.